code/modules/mob/living/silicon/robot/robot_vr.dm

```diff
@@ -70,3 +72,10 @@
 							cleaned_human << "<span class='warning'>[src] cleans your face!</span>"
 		return
 	return
+
+/mob/living/silicon/robot/proc/vr_sprite_check()
+	if(findtext(src.icon_state, "vr-", 1, 4) && (src.icon == 'icons/mob/robots.dmi'))
```

Comment:
... I dunno about this. This seems super janky. Another secret terrible thing like "all uniforms must end with _s or they don't work".